styles.css

```diff
@@ -34,6 +39,6 @@ img{
 
 @media(max-width:600px){
   .main{
-    background:red;
+  
```

Comment:
You need to make use of Flexbox's `order` property in here in order to rearrange the order of the requested HTML elements, per Question #7.